source/dubregistry/dbcontroller.d

```diff
@@ -472,6 +477,7 @@ struct DbPackage {
 	string[] categories;
 	long updateCounter = 0; // used to implement lockless read-modify-write cycles
 	@optional BsonObjectID logo; // reference to m_files
+	@optional string documentation_url;
```

Comment:
Naming convention -> `documentationURL`

Reply:
Cool. I was actually unsure about this as I saw many places which still used the underscore convention.